conda_smithy/templates/README.md.tmpl

```diff
@@ -14,6 +14,10 @@ Feedstock license: BSD-3-Clause
 
 Summary: {{ package.meta.about.summary }}
 
+{% if package.meta.about.dev_url %}Development: {{ package.meta.about.dev_url }}{% endif %}
```

Comment:
Isn't this adding unnecessary extra spaces when `dev_url` is not in the meta?

Reply:
Probably what do we need to fix that. This?

```suggestion
{% if package.meta.about.dev_url %}Development: {{ package.meta.about.dev_url }}{%- endif %}
```

Reply:
My change did add multiple new lines when no content was defined, so created https://github.com/conda-forge/conda-smithy/pull/1400

Reply:
Thanks :)